model.rb

```diff
@@ -97,7 +97,7 @@ def before_destroy
         end
       end
 
-      DeletedRecord.create(deleted_at: Time.now, model_name: model_name, model_values: model_values)
+      ArchivedRecord.create(deleted_at: Time.now, model_name: model_name, model_values: model_values)
```

Comment:
```suggestion
      ArchivedRecord.create(model_name: model_name, model_values: model_values)
```

I realize `deleted_at` was set before, but it's better to have the database set the default timestamp value.

Reply:
Agree. Applied your suggested change.